packages/astro/test/test-utils.js

```diff
@@ -151,7 +151,7 @@ export async function loadFixture(inlineConfig) {
 	};
 
 	const resolveUrl = (url) =>
-		`http://${config.server.host}:${config.server.port}${url.replace(/^\/?/, '/')}`;
+		`http://${config.server.host || 'localhost'}:${config.server.port}${url.replace(/^\/?/, '/')}`;
```

Comment:
Sometimes `config.server.host` can be `false`, so fallback to `localhost` in that case.